data/json/items/gun/762x25.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "id": "ppsh",
+    "type": "GUN",
+    "name": "PPSh-41",
+    "description": "The Soviet made PPSh-41, chambered in 7.62 Tokarev, provides a relatively large ammunition capacity coupled with low recoil and decent accuracy.",
+    "weight": 3630,
```

Comment:
I left the standard weight alone because it appears to be the unloaded weitht of the PPSh. If you have better weight data for the PPSh, post it and I will incluse it.